awx/main/tasks.py

```diff
@@ -489,7 +489,7 @@ def awx_isolated_heartbeat():
     # Slow pass looping over isolated IGs and their isolated instances
     if len(isolated_instance_qs) > 0:
         logger.debug("Managing isolated instances {}.".format(','.join([inst.hostname for inst in isolated_instance_qs])))
-        isolated_manager.IsolatedManager().health_check(isolated_instance_qs)
+        isolated_manager.IsolatedManager(CallbackQueueDispatcher.dispatch).health_check(isolated_instance_qs)
```

Comment:
The health check does actually *need* this function (because it doesn't emit job events), but if it ever _did_ for some reason, this is a close enough approximation (probably better than `None` or an empty `lambda`).